Update Zoo.cs

CCianfloneDev opened the pull request:
Impl Props.

CCianfloneDev committed:
Update Zoo.cs

Impl Props.

DZ-FSDev commented on ZooKeepingSystem/Zoo.cs:
```diff
@@ -7,68 +7,35 @@ namespace ZooKeepingSystem
-            this.numberOfZooKeepers = numberOfZooKeepers;
-        }
-        
-        /// <summary>
-        /// Initializes a zoo with a name, 0 cages, 1 zoo keeper, and no animals in the care.
-        /// </summary>
-        /// <param name="name">The name of the zoo.</param>
-        public Zoo(string name) : this(name, 0, 1, null)
-        {
-        }
```
- [x] This generic constructor should be restored.

DZ-FSDev commented on ZooKeepingSystem/Zoo.cs:
```diff
@@ -7,68 +7,35 @@ namespace ZooKeepingSystem
+        /// <param name="numberOfZooKeepers">Total number of zoo keepers employed.</param>
+        /// <param name="animals">Animals.</param>
+        public Zoo(int numberOfCages, int numberOfZooKeepers, Animals animals)
         {
-            this.numberOfZooKeepers = numberOfZooKeepers;
+            NumberOfCages = numberOfCages;
+            NumberOfZooKeepers = numberOfZooKeepers;
+            AnimalsInZoo = animals;
```
- [x] The `name` parameter should be included.

DZ-FSDev commented on ZooKeepingSystem/Zoo.cs:
```diff
@@ -7,68 +7,35 @@ namespace ZooKeepingSystem
     public class Zoo
     {
-        private Animals animals;
-        private string name;
```
- [x] We should retain the name of the Zoo; but as a property.

DZ-FSDev reviewed (changes requested):
**CR for #18**

CCianfloneDev committed:
Update Zoo.cs

Impl. Generic constructor, as well as added zooName as a parameter for a zoo object.

DZ-FSDev reviewed (approved):
#18 looks good!
